### PR TITLE
refactor: Move folders out of the /atoms folder

### LINKED ISSUE #7084: [Task]: Remove `badges`, `boxes`, `buttons`, ìconButtons`, `labels`, `pills` & `tiles` from `/atoms`

### PR DESCRIPTION
**Pull request by evavirseda**
## Summary

Closes #7084 

## Changelog

```
- Move badges, boxes, buttons, iconButtons, labels, pills, and tiles out of the /atoms folder
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by brancoder — APPROVED** [review]
LGTM

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
The app crashes in this branch in the following places:
- click send
- click search in tx history 
- click manage voting power
- ...

There must be some imports that are not updated and its causing the app to crash 🙏🏼

File: packages/desktop/components/popups/ReceiveAddressPopup.svelte [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
     import { QR, Text, FontWeight } from 'shared/components'
-    import { AddressBox } from 'shared/components/atoms'
+    import { AddressBox } from 'shared/components'
```
Comment (begonaalvarezd) [inline packages/desktop/components/popups/ReceiveAddressPopup.svelte:3]: lets get these imports together 🙏🏼 we can also use the alias `@ui` for the import `shared/components` which we should be using too I think

Comment (begonaalvarezd) [reply packages/desktop/components/popups/ReceiveAddressPopup.svelte:3]: there are couple appearances in this PR about these 2 points

**Review by VmMad — APPROVED** [review]

**Review by begonaalvarezd — APPROVED** [review]
🔥 🚒

**Pull request merged**